modules/admin/api/handler/cms_api_admin_handler.e

```diff
@@ -0,0 +1,74 @@
+note
+	description: "[
+			handler for CMS api rest admin in the CMS interface.
+		]"
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	CMS_API_ADMIN_HANDLER
```

Comment:
I don have problems I have used

CMS_API_ADMIN_<ROLE|USER>_HANDLER and you are suggesting to use
CMS_ADMIN_API_<ROLE|USER>_HANDLER. maybe the latest option is more close to
the current CMS names.




A